.github/ISSUE_TEMPLATE/10_model_request.yml

```diff
@@ -0,0 +1,19 @@
+name: Model request
+description: Request a new model for the library
+title: "MR: "
+labels: [mr]
```

Comment:
```suggestion
labels: ['model request']
```

Reply:
Fun fact: if you type `mr` + enter in the typeahead box when adding a label to an issue it will assign `model request`!